### PR TITLE
Add unattended upgrades

### LINKED ISSUE #7: Unattended Updates
I suggest we configure the studentenportal server to make unnatended updates autmatically.

See https://help.ubuntu.com/lts/serverguide/automatic-updates.html.en

### PR DESCRIPTION
**Pull request by fabianhauser**
Resolves #7

### CODE REVIEW

File: roles/base/tasks/unattended-upgrades.yml [hunk 1]
```diff
@@ -0,0 +1,26 @@
+---
+- name: Install unattended-upgrade packages
+  apt:
+      pkg: "{{ item }}"
```
[the remark this answers is not in the document]
Comment (fabianhauser) [reply roles/base/tasks/unattended-upgrades.yml]: I have no strong opinion about that, feel free to change it @mattbaumann 😉

**Comment by fabianhauser** [comment]
@mattbaumann Personally, I don't like sending emails if there is no strong reason to do so. In case something breaks, there are still logfiles that can be analysed :)

**Comment by fabianhauser** [comment]
@mattbaumann 

```
ERROR! Syntax Error while loading YAML.
  found unacceptable key (unhashable type: 'AnsibleMapping')

The error appears to have been in '/vagrant/roles/base/tasks/unattended-upgrades.yml': line 14, column 11, but may
be elsewhere in the file depending on the exact syntax problem.

The offending line appears to be:

  copy:
    src: {{ item.src }}
          ^ here
We could be wrong, but this one looks like it might be an issue with
missing quotes.  Always quote template expression brackets when they
start a value. For instance:

    with_items:
      - {{ foo }}

Should be written as:

    with_items:
      - "{{ foo }}"

Ansible failed to complete successfully. Any error output should be
visible above. Please fix these errors and try again.
```

**Comment by mattbaumann** [comment]
Thanks for testing, I brought vagrant up too and reproduced the error. Will send a fix in a few minutes. And excuse me for the erroneous commit.

**Comment by mattbaumann** [comment]
Because I made changes to this PR, I would like you to review the code too and approve it. Afterwards, feel free to rebase the code to master.

**Pull request merged**